mirrord-layer/src/socket.rs

```diff
@@ -129,6 +129,12 @@ const fn is_ignored_port(port: Port) -> bool {
     port == 0 || (port > 50000 && port < 60000)
 }
 
+/// Ignore localhost Ipv4 and Ipv6 addresses, used along with `is_ignored_port` for debuggers
+#[inline]
+fn is_ignored_ip(ip: IpAddr) -> bool {
```

Comment:
why not mix it with the `port_debug_patch`? those ips aren't generally ignored and might confuse reader